### PR TITLE
new feature

### PR DESCRIPTION
**Pull request by lakientri**
This is the new feature

### CODE REVIEW

File: webOpen.py [hunk 1]
```diff
@@ -3,3 +3,4 @@
 wb=webbrowser.open("https://code.visualstudio.com/docs/?dv=win")
 
 # hey its me
+# this is the new feature
```
Comment (lakientri) [inline webOpen.py:6]: Hello , I did this....

Comment (lenmorld) [reply webOpen.py:6]: looks good! 👍

File: webOpen.py [hunk 2]
```diff
@@ -3,3 +3,4 @@
 wb=webbrowser.open("https://code.visualstudio.com/docs/?dv=win")
```
Comment (lenmorld) [inline webOpen.py]: hey can you change this URL to google

**Review by lenmorld — CHANGES_REQUESTED** [review]
hey take a look at my comments

File: webOpen.py [hunk 3]
```diff
@@ -1,5 +1,6 @@
 # this is testing
 import webbrowser
-wb=webbrowser.open("https://code.visualstudio.com/docs/?dv=win")
+wb=webbrowser.open("www.google.com")
```
Comment (lenmorld) [inline webOpen.py:3]: 👍

**Review by lenmorld — APPROVED** [review]
LGTM!

**Pull request merged**